src/Pos/Ssc/Class/Storage.hs

```diff
@@ -43,17 +42,21 @@ class Ssc ssc =>
       SscGStateClass ssc where
     -- | Load global state from DB by recreating it from recent blocks.
     sscLoadGlobalState
-        :: (HasContext LrcContext m, MonadDB m, WithLogger m)
+        :: (MonadDB m, WithLogger m)
         => m (SscGlobalState ssc)
+    -- | Dump global state to DB.
+    sscGlobalStateToBatch :: SscGlobalState ssc -> Tagged ssc [SomeBatchOp]
```

Comment:
What is the reason you use `Tagged` here? `ssc` is used in type declaration of the function, so it shouldn't be the problem (i guess).